ops/main.py

```diff
@@ -96,6 +106,9 @@ def _setup_event_links(charm_dir, charm):
     charm -- An instance of the Charm class.
 
     """
+    # XXX: on windows this is almost certainly completely broken
```

Comment:
What is almost certainly broken? I mean, what is "this" in that sentence? The `link_to` creation? Or the whole function?

Reply:
good point, i'll clarify. The thing that the method is trying to do won't work on Windows (I thought the rest of the comment explained that). The method does what it's supposed to do, but that won't result in working hooks.